new/LOC7368Tiberias.xml

```diff
@@ -0,0 +1,53 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LOC7368Tiberias" xml:lang="en" type="place">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ጥብርያዶስ</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-02-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="lake">
+                    <placeName>ጥብርያዶስ</placeName>
```

Comment:
In case the record remains this must be corrected extended according to the Guidelines https://betamasaheft.eu/Guidelines/?q=wikidata&id=places, see example 3, with the transcription, translation, (English marked as main), sameAs should be pointing to wikidata entry (and BM ID added to wikidata after the entry has been corrected and merged), location pointing at least to Israel/Palestine and better also to the region LOC3221Galile

```        
<place type="lake" sameAs="wd:Q126982">
              <placeName xml:id="n1" xml:lang="gez">ጥብርያዶስ</placeName>
              <placeName corresp="#n1" xml:lang="gez" type="normalized">Ṭəbəryās</placeName>
              <placeName corresp="#n1" xml:lang="en" type="main">Tiberias</placeName>
              <placeName xml:id="n2" xml:lang="en">Sea of Galilee</placeName>
              <placeName xml:id="n3" xml:lang="en">Kinneret</placeName>

              <country ref="LOC3937Israel"/> (or rather LOC5068Palest?)
              <region ref="LOC3221Galile"/>
</place>
```

Reply:
But what for is ID `LOC5969Tiberi`?

Reply:
Searching for Tiberias on Betamasaheft.eu leads to LOC5969 and this leads to Q151920, what is supposed to be the city of Tiberias.

 <html>
<body>
<!--StartFragment-->

Q151920
--


<!--EndFragment-->
</body>
</html>[Q151920](https://www.wikidata.org/wiki/Q151920)

Reply:
I cannot find LOC7368Tiberias in Oxygen. I switched the branch in Github to (Places->CH_2023-02-01_Tiberias and fetched), but I do not see the record in Oxygen.

Reply:
Here it is already displayed: https://betamasaheft.eu/places/LOC7368Tiberias/main

Reply:
I found it in the Desktop folder.

Reply:
Done.

Reply:
Entries created online as stubs are always displayed on the app. Also the entries which you wanted to create but we decided we do not need (Yosef, Matewos, Kerubim) are currently displayed, I have to delete them there first.